packages/orbit-components/src/Text/helpers/getMargin.ts

```diff
@@ -0,0 +1,34 @@
+import type { Props } from "../types";
+
+const getMargin = (margin: Props["margin"]): { vars: object; classes: string[] } => {
+  if (typeof margin === "string")
```

Comment:
Instead of actually conditionally returning it, you can create the helper function that will do it for you and just call it here inside :)

Reply:
I am not sure I understand. The helper function would be defined outside this one but still on the context of Text? I don't think it will be reused anywhere, so what would be the benefit here? 🤔

Reply:
What I meant is something like that: 

```
const getMarginValue = (margin: string | number): string =>
  typeof margin === "number" ? `${margin}px` : margin;

const getMargin = (margin: Props["margin"]): { vars: object; classes: string[] } => {
  if (typeof margin === "string" || typeof margin === "number")
    return {
      vars: { "--text-margin": getMarginValue(margin) },
      classes: ["m-[var(--text-margin)]"],
    };

  const { top, bottom, left, right } = margin || {};
  const cssVar = {};
  const marginClasses: string[] = ["m-0"];

  if (top) {
    marginClasses.push("mt-[var(--text-margin-top)]");
    cssVar["--text-margin-top"] = getMarginValue(top);
  }

  if (bottom) {
    marginClasses.push("mb-[var(--text-margin-bottom)]");
    cssVar["--text-margin-bottom"] = getMarginValue(bottom);
  }

  if (left) {
    marginClasses.push("ml-[var(--text-margin-left)]");
    cssVar["--text-margin-left"] = getMarginValue(left);
  }

  if (right) {
    marginClasses.push("mr-[var(--text-margin-right)]");
    cssVar["--text-margin-right"] = getMarginValue(right);
  }

  return { vars: cssVar, classes: marginClasses };
};
```

Basically just a suggestion just DRY, as you are repeating those condition on multiple places

Reply:
Got it, agreed. Will refactor into that